clab/clab.go

```diff
@@ -123,11 +124,17 @@ func WithKeepMgmtNet() ClabOption {
 	}
 }
 
-func WithTopoFile(file, varsFile string) ClabOption {
+func WithTopoRef(topoRef, varsFile string) ClabOption {
```

Comment:
How about `WithTopoPath`? A path can be all the way to the file or to the dir

Reply:
done